phonon/process.py

```diff
@@ -78,13 +80,12 @@ def __init__(self, process_ttl=int(0.5*TTL), host='localhost', port=6379, db=1,
         self.__heartbeat_ref = None
 
         if not hasattr(Process, 'client'):
-            Process.client = redis.StrictRedis(host=host, port=port, db=db)
+            if hosts is not None:
+                Process.client = ShardedClient(hosts=hosts, port=port, db=db)
+            else:
+                Process.client = redis.StrictRedis(host=host, port=port, db=db)
```

Comment:
The only reason we wouldn't would be a user's expectation that they have a `StrictRedis` vs a `ShardedClient` in older implementations. You could totally argue older implementations just shouldn't upgrade. I guess the answer is; there isn't _really_ a good reason not to.